src/lib.rs

```diff
@@ -114,7 +125,7 @@ pub fn render_pdf() -> Vec<u8> {
     let document = compile();
     let world = world();
 
-    typst_pdf::pdf(&document, Some(""), world.today(Some(0)))
+    typst_pdf::pdf(&document, Smart::Auto, world.today(Some(0)))
```

Comment:
I have no idea about smart pointers and just chose "Auto" because it sounded good. Second look appreciated.